src/test/resources/ldap/elytron-keystore-tests.ldif

```diff
@@ -0,0 +1,124 @@
+
+dn: ou=keystore,dc=elytron,dc=wildfly,dc=org
+objectclass: top
+objectclass: organizationalUnit
+ou: keystore
+
+dn: cn=firefly,ou=keystore,dc=elytron,dc=wildfly,dc=org
```

Comment:
Looking at the name I assume the content here is based on the Elytron certificate authority within the project, it may be nice if we could have a Readme near here to confirm this and some information on how to concert from the CA files to the LDIF so if things are re-generated it will not take much to update this as well.